SPEC.md

```diff
@@ -765,7 +765,7 @@ JSON Schema for the Image Manifest (app image manifest, ACI manifest), conformin
     * **app** (string, required) name of the dependent App Container Image.
     * **imageID** (string of type [Image ID](#image-id-type), optional) content hash of the dependency. If provided, the retrieved dependency must match the hash. This can be used to produce deterministic, repeatable builds of an App Image that has dependencies.
     * **labels** (list of objects, optional) a list of the very same form as the aforementioned label objects in the top level ImageManifest. See [Dependency Matching](#dependency-matching) for how these are used.
-* **pathWhitelist** (list of strings, optional) complete whitelist of paths that may exist in the rootfs after assembly (i.e. unpacking the files in this image and overlaying its dependencies, in order). If a path ends in a slash, then the ACE must ensure that the directory is empty; if the image or a dependency contains files beneath this path, they will not be present in the rendered filesystem. This field is only required if the app has dependencies and you wish to remove files from the rootfs before running the app. An empty list is equivalent to an absent value and means that all files in this image and any dependencies will be available in the rootfs.
+* **pathWhitelist** (list of strings, optional) complete whitelist of paths that may exist in the rootfs after assembly (i.e. unpacking the files in this image and overlaying its dependencies, in order). Paths must be absolute and are not recursive. If a path ends in a slash, then the ACE must ensure that the directory is empty - i.e., if the image or a dependency contains files beneath this path, they will not be present in the rendered filesystem. This field is only required if the app has dependencies and you wish to remove files from the rootfs before running the app. An empty list is equivalent to an absent value and means that all files in this image and any dependencies will be available in the rootfs.
```

Comment:
One more nit: "If a path ends in a slash" - what happens if I list a path that is a directory but I do not put a slash?  You already said dirs are not recursive.  Maybe this whole clause is moot?